tests/functional/b/bad_builtins.txt

```diff
@@ -0,0 +1,4 @@
+bad-builtin:2:::Used builtin function 'input'
+bad-builtin:3:::Used builtin function 'filter'. Using a list comprehension can be clearer.
+bad-builtin:3:::Used builtin function 'map'. Using a list comprehension can be clearer.
+bad-builtin:3:::Used builtin function 'print'
```

Comment:
```suggestion
bad-builtin:2:0::Used builtin function 'input'
bad-builtin:3:15::Used builtin function 'filter'. Using a list comprehension can be clearer.
bad-builtin:3:6::Used builtin function 'map'. Using a list comprehension can be clearer.
bad-builtin:3:0::Used builtin function 'print'
```
The column numbers are missing. I'm wondering though why only the Windows tests and then only 3.8 and 3.9 are failing. @Pierre-Sassoulas Do you have an idea?

Reply:
We only check the column number on 3.8 plus because the number are more relevant on 3.8. I can generate them for you if your interpreter is not 3.8

Reply:
👍🏻 I was also / mainly interested in the reason why the linux tests didn't fail. Seems like the line numbers aren't relevant there, too.